src/error.rs

```diff
@@ -16,6 +16,8 @@ pub enum Error {
     MultipartRejection(#[from] axum::extract::multipart::MultipartRejection),
     #[error("Missing filename multipart")]
     MissingFilename,
+    #[error("Unsupported file format: {0}")]
```

Comment:
I wonder if we should provide a list of supported file formats. Not 100% necessary.

Reply:
Yeah, could be. Also the frontend should refuse to send unsupported files